pmd-java/src/test/java/net/sourceforge/pmd/lang/java/rule/performance/AvoidInstantiatingObjectsInLoopsTest.java

```diff
@@ -4,8 +4,98 @@
 
 package net.sourceforge.pmd.lang.java.rule.performance;
 
-import net.sourceforge.pmd.testframework.PmdRuleTst;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
 
-public class AvoidInstantiatingObjectsInLoopsTest extends PmdRuleTst {
```

Comment:
You have to keep the base class `PmdRuleTsts` - this provides all the magic to execute the rule test cases. Now you disabled a bunch of tests...
More background info: https://pmd.github.io/latest/pmd_userdocs_extending_testing.html